ipaullly/sendIT
# [#161951629] Update README.md to include instructions for running flask app locally

#### What does this PR do?
Adds instructions and specific cmd lines to the README.md so that whoever sees it can easily run the app on their local machine 
#### Description of Task to be completed?

- Add commands to set `FLASK_APP` and `FLASK_ENV`

- Add command to initialize flask

- Update POSTMAN instructions to include how to power it up for testing

#### How should this be manually tested?
go to the GitHub repo https://github.com/ipaullly/sendIT/tree/ch-local-app-commands-161951629 and scroll down the README.md to see updated instructions on how to run the sendIT flask app locally.
